packages/build/bin/lint-commits.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
This file doesn't need to be moved as we don't expose it in `build` and `loopback-next` still seems to be using the one from `/bin`.